integration/docker_integration_test_executor.go

```diff
@@ -80,16 +81,32 @@ func givenThisContainer(t *testing.T, bundle IntegrationTestDockerExecRunnerBund
 		t.Fatalf("")
 	}
 
+	params := []string{"run", "--detach", "-v", localPiper + ":/piper", "--name=" + testRunner.ContainerName}
+	if testRunner.User != "" {
+		params = append(params, fmt.Sprintf("--user=%s", testRunner.User))
+	}
+	if len(bundle.TestDir) > 0 {
+		projectDir := path.Join(wd, path.Join(bundle.TestDir...))
+		params = append(params, "-v", fmt.Sprintf("%s:/project", projectDir))
```

Comment:
The handling of tempDir happens as before, but should be adapted the check for `len(bundle.TestDir)` originally introduced in this change. I.e. tempDir is created and filled unconditionally, and then ignored be the above two lines.

Reply:
right. the merge was a bit complex, this went wrong. thanks.